tests/junit.py

```diff
@@ -109,6 +119,10 @@ def check_pre_skip(self, test: str, spec: TestSpec, resource: str, nproc: int) -
         for condition in spec.only:
             if (condition == 'cpu') and ('gpu' in resource):
                 return 'CPU only test with GPU backend'
+            if (condition == 'int32') and ('int64' in self.petsc_arch):
+                return 'int32 test with int64 PETSc arch'
+            if (condition == 'int64') and ('int64' not in self.petsc_arch):
+                return 'int64 test with int32 PETSc arch'
```

Comment:
This presumes that everyone would have `int64` or similar in their `PETSC_ARCH` names. Is there a better way to verify the 64bit-nature of a PETSc build?

Reply:
Using Jed's comment above